seed-server/network-connection.sh

```diff
@@ -29,16 +29,27 @@ fileNodeNetwork=
 [ -e "${baseConfigDir}/podNetwork" ] && filePodNetwork=$(cat ${baseConfigDir}/podNetwork)
 [ -e "${baseConfigDir}/nodeNetwork" ] && fileNodeNetwork=$(cat ${baseConfigDir}/nodeNetwork)
 
-n=123
 is_ha=
-if [[ $POD_NAME =~ .*-([0-4])$ ]]; then
+if [[ $POD_NAME =~ .*-([0-2])$ ]]; then
```

Comment:
So we are now only expect up to three instances. Right?

Reply:
Right. Using  `/26` ranges restricts the possible number of instances to three. The fourth range is needed on the client side for the network for the bonding device.